docs/geode-native-docs/continuous-querying/3-native-client-cq-api.html.md.erb

```diff
@@ -21,9 +21,10 @@ limitations under the License.
 
 This chapter documents the primary native client API for CQ management.
 
-The Apache Geode native client API allows your clients to create and manage CQs. (The server side does not have an API.) Continuous querying provides native client query syntax, events-based management, integration with client/server architecture, active query execution, and interest criteria based on data values. For complete information on the classes and interfaces described here, see the API documentation at [http://gemfire-apis.docs.pivotal.io](http://gemfire-apis.docs.pivotal.io).
+The Apache Geode native client API allows your clients to create and manage CQs. (The server side does not have an API.) Continuous querying provides native client query syntax, events-based management, integration with client/server architecture, active query execution, and interest criteria based on data values. For complete information on the classes and interfaces described here, 
+see the API documentation for [C++](/releases/latest/api/cppdocs) or [C\#](/releases/latest/api/dotnetdocs).
```

Comment:
Would it be better to use ".NET" instead of "C#"  (in both files with these links)?